src/generate-projects-json.mjs

```diff
@@ -33,15 +34,16 @@ const projects = yamls.map((path) => {
   }
   if (parsedYaml.tutorials) {
     parsedYaml.tutorials = parsedYaml.tutorials.filter((t) =>
-      sanitizeUrl(Object.values(t)[0]),
+      sanitizeUrl(Object.values(t)[0])
     );
   }
 
   return parsedYaml;
 });
 
-writeFileSync(
-  './public/projects.json',
-  JSON.stringify(projects, null, 4),
-  'utf8',
-);
+mkdirSync("./dist", { recursive: true });
+const outputPaths = ["./public/projects.json", "./dist/projects.json"];
```

Comment:
When running `vite build` you can change `--outDir` to something else.

Let's add a CLI parameter to `generate-projects-json.mjs` for the projects.json file path. By default it can be `./dist/projects.json` if none provided.